### DISCUSSION TITLE
Snippets

### DISCUSSION THREAD

**Discussion by antoniorws** [dbeaver/dbeaver#17829] [DBeaver Development]
How to create snippets in dbeaver?
Is there this option?

**Comment by LonwoLonwo** [comment]
Hello @antoniorws 
What kind of snippets do you searching for?
Provide some examples, please.

**Reply by antoniorws** [reply]
Hi @LonwoLonwo, I don't know if you know, but in SQL Developer I can save some SQL Instructions and when I need, I just drag and drop.
Looks like with it:
![image](https://user-images.githubusercontent.com/18173049/194567915-b7f49881-bc80-4b7a-9d6b-61b4772eafc2.png)
![image](https://user-images.githubusercontent.com/18173049/194568108-d358242a-16ea-4079-8e56-f40cf6d94e8d.png)
![image](https://user-images.githubusercontent.com/18173049/194568145-88404eec-c552-4db1-9fd2-f5f01fcd6b13.png)

**Reply by LonwoLonwo** [reply]
I think you can use templates as snippets:

![2022-10-11 04_17_17-DBeaver Ultimate 22 2 2 - _postgres 13 3_Script-169](https://user-images.githubusercontent.com/45152336/194976198-7ea27d1e-01a9-4116-9248-c62527ba1fd7.png)

![2022-10-11 04_17_50-DBeaver Ultimate 22 2 2 - _postgres 13 3_Script-169](https://user-images.githubusercontent.com/45152336/194976218-0836fd6c-c115-4aa0-820b-0ed0b09d00f5.png)

**Reply by antoniorws** [reply]
Thank you, @LonwoLonwo :D
